dashboard/pkg/epinio/dialog/LoginDialog.vue

```diff
@@ -0,0 +1,199 @@
+<script>
+import GenericPrompt from '@shell/dialog/GenericPrompt';
+import Tabbed from '@shell/components/Tabbed/index.vue';
+import Tab from '@shell/components/Tabbed/Tab.vue';
+import epinioAuth, { EpinioAuthTypes } from '../utils/auth';
+import Password from '@shell/components/form/Password';
+import { LabeledInput } from '@components/Form/LabeledInput';
+
+export default {
+  name: 'LoginDialog',
+
+  components: {
+    GenericPrompt, Tabbed, Tab, LabeledInput, Password
+  },
+
+  props: {
+    resources: {
+      type:     Array,
+      required: true
+    }
+  },
+
+  data() {
+    return {
+      selectedTab: '',
+      username:    '',
+      password:    '',
+      config:      {
+        applyMode:   'login',
+        applyAction: this.login,
+      },
+      tab: {
+        local: 'local',
+        dex:   'dex'
+      }
+    };
+  },
+
+  computed: {
+    cluster() {
```

Comment:
Is there a specific reason to define `cluster` as computed value ?

Reply:
Just for readability. This makes it clearer that it's a cluster, instead of an odd first entry in an array